contrib/fine-tuning/fine-tune-mlx.sh

```diff
@@ -7,23 +7,29 @@ MAX_SEQ=262144
 MAX_TOKENS=16384
 ITERS=1500
 NUM_LAYERS=48
+ADAPTERS_PATH=adapters
+TEMP=0.7
 case "$TOOL_BASE_MODEL" in
   cdx1-mini)
+    ADAPTERS_PATH=adapters-mini
     NUM_LAYERS=36
     BASE_MODEL="unsloth/Qwen3-4B-Instruct-2507"
     ;;
   cdx1-pro)
+    ADAPTERS_PATH=adapters-pro
     ITERS=2500
     BASE_MODEL="unsloth/Qwen3-Coder-30B-A3B-Instruct"
     ;;
   *)
+    TEMP=0.55
     ITERS=2000
+    MAX_SEQ=32768
```

Comment:
We had previously fine-tuned 16 layers with 16K tokens, which confused it to the core.